pkg/splunk/common/urls.go

```diff
@@ -10,87 +10,20 @@ const (
 	//ClusterManager = "cluster-master"
 	ClusterManager = "cluster-master"
 
```

Comment:
These APIs are now supported by Splunk ES with the `/manager/*` so they are no longer needed here.